### PR TITLE
Add disabled dates section

### PR DESCRIPTION
**Pull request by zdrawku**

### CODE REVIEW

File: en/components/calendar.md [hunk 1]
```diff
@@ -234,6 +234,101 @@ Having implemented this conditional templating and date parsing we should get co
...
+This section demonstrates the usage of `disabledDates` functionallity. Different `single dates` or `range` elements could be added to Array, and passed to the `disabledDates` descriptor.
+
+```typescript
+this.calendar.disabledDates = [{ type: DateRangeType.Between, dateRange: [
+    new Date(2018, 8, 2),
+    new Date(2018, 8, 8)
+])];
+```
+
+The `DateRangeType` is used to specify the range that is going to be disabled. For example, `DateRangeType.Between` will disable the dates between two specific dates in Array. Code snipped above.
```
Comment (tiliev) [inline en/components/calendar.md]: Should be `snippet` instead of `snipped`.

File: en/components/calendar.md [hunk 2]
```diff
@@ -263,6 +358,8 @@ When a day inside the current month is focused:
 | `formatOptions` | `Object` | The format options passed along with the `locale` property used for formatting the dates. Defaults are { day: 'numeric', month: 'short', weekday: 'short', year: 'numeric' }. |
 |`formatViews`| `Object`| Controls whether the date components in the different calendar views should be formatted according to the provided locale and formatOptions. Defaults are { day: false, month: true, year: false }. Does not affect rendering in the header. |
 | `vertical` | `boolean` | Controls the layout of the calendar component. When vertical is set to true the calendar header will be rendered to the side of the calendar body.|
+| `disabledDates` | `array` | Gets\Sets the disabled dates descriptors. |
```
Comment (tiliev) [inline en/components/calendar.md]: Should be `Gets/Sets` instead of `Gets\Sets`.

File: en/components/date_picker.md [hunk 3]
```diff
@@ -218,6 +218,8 @@ The result is as follows:
 | `formatViews` | `Object` | The format views passed along with the `locale` property used for formatting dates. |
 | `label` | `string` | Changes the default text label. |
 | `labelVisibility` | `string ` | Hide/Show the label. |
+| `disabledDates` | `array` | Gets\Sets the disabled dates descriptors. |
```
Comment (tiliev) [inline en/components/date_picker.md]: Should be `Gets/Sets` instead of `Gets\Sets`.

File: en/components/calendar.md (hunk 1 above)
Comment (zdrawku) [reply en/components/calendar.md]: Done

File: en/components/calendar.md (hunk 2 above)
Comment (zdrawku) [reply en/components/calendar.md]: Done

File: en/components/date_picker.md (hunk 3 above)
Comment (zdrawku) [reply en/components/date_picker.md]: Done

**Pull request merged**